protos/guard/Guard.proto

```diff
@@ -0,0 +1,192 @@
+syntax = "proto3";
+
+
+
+package btfs;
+
+
+
+service GuardService{
+	rpc SubmitFileStoreMeta(FileStoreStatus) returns (Result){}
+	//rpc health() returns(HealthStatus){} //used for other nodes check guard health , TBD input argument discuss with Weiyu & Jin
+  	rpc SendQuestions(ChallengeQuestions) returns(Result){} //used for renter send the questions to guard
+  	rpc CheckFileStoreMeta(CheckFileStoreMetaRequest) returns (FileStoreStatus){} //reserved for other nodes to check contract status
+}
+
+message Timestamp {
```

Comment:
Escrow PR was using google grpc's timestamp? Could you sync to be the same?

Reply:
Done the change. Actually my timestamp was copied from google too, it is good for can just use reference.